contracts/mainnet/connectors/permit_erc20/main.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity ^0.7.6;
+pragma experimental ABIEncoderV2;
+
+import {TokenInterface, MemoryInterface} from "../../common/interfaces.sol";
+import {Stores} from "../../common/stores.sol";
+import {TokenInterfaceWithPermit, DAITokenInterfaceWithPermit} from "./interface.sol";
+//import {Helpers} from "./helpers.sol";
+import {Events} from "./events.sol";
+
+/**
+ * @title ERC20 Permit.
+ * @dev Deposit ERC20 using Permit.
+ */
+
+
+contract ERC20PermitResolver is Stores {
+    address private immutable daiAddress = 0x6B175474E89094C44Da98b954EedeAC495271d0F; // dai has a different implementation for permit
```

Comment:
```suggestion
    address constant immutable daiAddress = 0x6B175474E89094C44Da98b954EedeAC495271d0F; // dai has a different implementation for permit
```

Reply:
Gives a compiling error when we use constant and immutable together here @pradyuman-verma

Reply:
@harshit-wq  use `constant internal`

Reply:
done